data-reset-tool/test/postman/loader.postman_collection.json

```diff
@@ -0,0 +1,95 @@
+{
+	"info": {
+		"_postman_id": "ab4cde71-ade5-4fc4-87a3-83e74d7bb120",
+		"name": "loader",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "get",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "6a68e659-28b7-439c-abce-a4a5e084df93",
+						"exec": [
+							"pm.environment.set(\"binary_response\", pm.response.stream);"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"protocolProfileBehavior": {
+				"disableBodyPruning": true
+			},
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "file",
+					"file": {}
+				},
+				"url": {
+					"raw": "http://localhost:5000/api/fixture/export/CP0001333/excel",
+					"protocol": "http",
+					"host": [
+						"localhost"
+					],
+					"port": "5000",
+					"path": [
+						"api",
+						"fixture",
+						"export",
+						"CP0001333",
+						"excel"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "post",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "Content-Type",
+						"name": "Content-Type",
+						"value": "multipart/form-data",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "file",
+							"type": "file",
+							"src": "/Users/kjinnah/Downloads/CP0001333.xls"
```

Comment:
should this be hard-coded? Same for the localhost:5000 above

Reply:
I think it's fine for now. No matter what you'll need to change the path to the excel file manually until John creates the scenarios he wants and checks them in. The URL will change once Katie creates the deployment and adds a dev/test/prod url, but I think that should be part of her ticket

Reply:
I guess I could check in he excel file I was using and point it to that for now though